pkgs/development/tools/language-servers/nixd/default.nix

```
@@ -129,6 +129,13 @@ in
         boost182
       ];
 
+      nativeBuildInputs = [
```

```suggestion
      nativeBuildInputs = common.nativeBuildInputs ++ [ llvmPackages.llvm ];
```

Maybe better just declaring what is need for cross compilation.